pkg/apis/eksctl.io/v1alpha5/validation.go

```diff
@@ -433,10 +435,19 @@ func validateNodeGroupLabels(labels map[string]string) error {
 	return nil
 }
 
+func isKubernetesRecommendedLabel(namespace string) bool {
+	for _, domain := range []string{"app.kubernetes.io"} {
+		if namespace == domain {
+			return true
+		}
+	}
+	return false
```

Comment:
does this need to be a loop right now given we only have the one domain? we can add the loop back when we need it I think.

```suggestion
	return namespace == domain
```